// ==========================================================================
// Copyright (C) 2013 Portuguese Air Force Academy                       =
// Research Center                                                          =
// Granja do Marques, Academia, Pero-Pinheiro                               =
// ==========================================================================
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
// 
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
// GNU General Public License for more details.
// 
// You should have received a copy of the GNU General Public License
// along with this program; if not, write to the Free Software Foundation,
// Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
// ==========================================================================
// Project: piccolo_ros-Debug@piccolo_ros
// Date:    10:45:51 PM                                
// Author:  posilva                                
// -------------------------------------------------------------------------=
// Change Log:                                                         
// -------------------------------------------------------------------------=
// Date       / Author              /Description                          
// ==========================================================================
// Description: PiccoloRosMessages.h
// 
// ==========================================================================

#ifndef PICCOLOROSMESSAGES_H_
#define PICCOLOROSMESSAGES_H_
#include <piccolo_ros/PiccoloTelemetry.h>
#include <piccolo_ros/PiccoloCommand.h>
#include <piccolo_ros/PiccoloTrack.h>
#include <piccolo_ros/PiccoloStatus.h>
#include <piccolo_ros/PiccoloWPList.h>
#include <piccolo_ros/PiccoloWaypoint.h>
#include <piccolo_ros/PiccoloToPayload.h>
#include <piccolo_ros/PiccoloFromPayload.h>
#include <piccolo_ros/PiccoloRequestWaypoints.h>
#endif /* PICCOLOROSMESSAGES_H_ */
